saleor/order/actions.py

```diff
@@ -83,6 +88,8 @@ def cancel_order(order: "Order", user: Optional["User"]):
     manager.order_cancelled(order)
     manager.order_updated(order)
 
+    send_order_canceled_confirmation(order, user)
```

Comment:
Not a big issue but sometimes we say "cancelled" and sometimes "canceled" - both are correct but we have them mixed in entire codebase. Do you have any opinion on which one should we use @patrys?

Reply:
"Cancelled" is British, "canceled" is American.

Reply:
Yeah I know, but do we have any preference on which to use?

Reply:
Looking at the code it seems American English is used almost everywhere (including the code itself).